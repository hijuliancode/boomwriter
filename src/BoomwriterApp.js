import React from 'react';
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
import styled from 'styled-components';

import HomeComponent from "./components/Home";
import TheWritingBeeComponent from "./components/TheWritingBee";
import HeaderComponent from "./components/shared/Header";
import FooterComponent from "./components/shared/Footer";

const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  height: 100%;
`;

const BoomwriterApp = () => {
  return (
    <Router>
      <Wrapper>
        <HeaderComponent />
        <Switch>
          <Route exact
            path="/home"
            component={() => <HomeComponent />} />
          <Route exact
            path="/the-writing-bee"
            component={() => <TheWritingBeeComponent />} />
        </Switch>
        <FooterComponent />
      </Wrapper>
    </Router>
  );
}

export default BoomwriterApp;
