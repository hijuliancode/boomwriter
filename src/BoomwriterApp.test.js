import React from 'react';
import { shallow } from 'enzyme';
import BoomwriterApp from './BoomwriterApp';

describe('test in BoomwriterApp', () => {
  test('renders BoomwriterApp', () => {
    const wrapper = shallow(<BoomwriterApp />);
    expect( wrapper ).toMatchSnapshot();
  });
})

