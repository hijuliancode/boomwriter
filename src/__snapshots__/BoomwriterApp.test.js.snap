// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in BoomwriterApp renders BoomwriterApp 1`] = `
<div
  className="BoomwriterApp"
>
  BoomwriterApp works!
</div>
`;
